submissions/expero1/js-dom/script.js

```diff
@@ -0,0 +1,82 @@
+const createElem = (elem, attrObj, className) => {
+  elem = document.createElement(elem);
+  elem.classList.add(className);
+  for (attrName in attrObj) {
+    elem.setAttribute(attrName, attrObj[attrName]);
+  }
+  return elem;
+};
+
+const createMenu = (data) => {
+  let menu = createElem("ul", { id: "menu", class: "menu" });
+  data.forEach(
+    (item, index) => { menu.appendChild(createMenuItem(data[index], index)) }
```

Comment:
You can use your `item` instead of `data[index]`
```suggestion
    (item, index) => { menu.appendChild(createMenuItem(item, index)) }
```

Reply:
My fault. Thanks. Fix.